evm-contracts/src/v0.6/dev/AnswerValidatorInterface.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity 0.6.6;
+
+interface AnswerValidatorInterface {
+  function validate(
```

Comment:
Consider splitting into two functions: a `isValid` view, and a `validate` function. Also, a comment explaining what should be expected usage/result of the `validate` function would be great.

Reply:
This will enable easier testing of Validator implementations using the `isValid` function. Also, external agents could check the validity of the answers by using this `view` function without spending gas unnecessarily for the `validate` function.

Reply:
I did not add it to the required interface, but I did break out an `isValid` function to more easily test it. I figure we can expand the interface if it proves to be useful, but for now we should keep it small until we see more concrete implementations and are understand what we really need.